test/base_test.py

```diff
@@ -593,7 +593,8 @@ def test_set_error(self):
                         # FluidSynth support. Setting environment variable
                         # SDL_SOUNDFONTS to the path of a valid sf2 file
                         # removes the error message.
-                        e == "No SoundFonts have been requested",
+                        e == "No SoundFonts have been requested" or
+                        e.startswith("Failed to access the SoundFont"),
```

Comment:
AFAICT, SDL_mixer seems has changed it's behaviour since this pygame test was written. Both pygame-cffi and pygame end up with a "Failed to access the SoundFont <path>" when run on my various machines, but I do get the "No Soundfonts have been requested" from an older SDL installtion in a chroot.  Since this is a different version of the non-fatal failure the "No SoundFonts" check is there for, we should also ignore it.

The path in the error message, however, seems to be relative to the SDL_mixer installation, so including it in the check looks like a bad idea.